src/View/Form/EntityContext.php

```diff
@@ -230,6 +251,26 @@ public function val($field)
     }
 
     /**
+     * Get default value from table schema for given entity field.
+     *
+     * @param string $field Field name.
+     * @param \Cake\Datasource\EntityInterface $entity The entity.
+     * @return mixed
+     */
+    protected function _schemaDefault($field, $entity)
+    {
+        $table = $this->_getTable($entity);
+        if ($table === false) {
+            return null;
+        }
+        $defaults = $table->schema()->defaultValues();
```

Comment:
What do you think about having a method to fetch the default value of a field? `$schema->defaultValue($column);` as an example.